assets/js/main.js

```diff
@@ -0,0 +1,167 @@
+(function() {
+  // ================== globals ==================
+  var COLORS = ['red', 'green', 'blue', 'yellow'];
+  var START_DIFFICULTY = 3;
+  var COLOR_LIGHT_TIME = 300;
+  var STARTING_TIME_PER_COLOR = 1000;
+
+  var sounds = {};
+
+  var $colors = {};
+  var $simon = null;
+  var $score = null;
+
+  var game = null;
+
+  // ================== utility ==================
+  // Returns a random integer between min (included) and max (excluded)
+  // Using Math.round() will give you a non-uniform distribution!
+  var getRandomInt = function(min, max) {
```

Comment:
I'm OK with either. I personally like using function functionName() {} for my own code, but it's a stylistic choice. We should pick one and use it consistently for the course. 

The BIG upside to var functionName = function()...is that it explicitly teaches students that 'functions are data/values' like everything else. you can store functions in variables, and pass them as arguments, just like you can with every other type of value.